prometheus_hardware_exporter/collector.py

```diff
@@ -547,14 +548,17 @@ def fetch(self) -> List[Payload]:
                     "Unknown ipmi SEL state: %s. Treating it as Nominal.", sel_entry["State"]
                 )
                 sel_state_value = sel_states_dict["NOMINAL"]
+            sel_entries_tuples.add((sel_entry["Name"], sel_entry["Type"], sel_state_value))
+
+        for sel_item in sel_entries_tuples:
```

Comment:
not-blocking: suggestion
```suggestion
        for sel_name, sel_type, sel_state_value in sel_entries_tuples:
```